ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml

```
@@ -15,21 +15,22 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- context_dir: images
-  dockerfile_path: Dockerfile
+- dockerfile_path: images/Dockerfile
```

```suggestion
- dockerfile_path: Dockerfile
```

@jkhelil you'll have to refactor https://github.com/openshift/csi-driver-shared-resource/pull/92 so that the dockerfiles live in the repo's top level dir

---

this is now fixed